README.md

```diff
@@ -1,2 +1,81 @@
-# react-capstone-project
-This is the react capstone project
+
+#BookStore
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+## Screenshot
+
+![screenshot](./a.png)
+
+![screenshot](./b.png)
+
+## Live Link
+[Click Here](https://astounding-kitsune-d200f4.netlify.app/)
+
+## Loom video demo 
+[Click here](https://www.loom.com/share/34c819273ec14dd1804fd1a5ca49c624)
+
+## Built With
+- Major Languages: JAVASCRIPT, HTML, CSS
+- Major Technologies : Git, Github, Webpack, React, Redux
+
+### Prerequisites
+- npm ( Node Package Manager )
+- git and Github Account
+
+
+### Install linters
+- **Webhint** linter
+
+  `npm install --save-dev hint@6.x`
+
+- **Stylelint** linter
+
+  `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x`
+
+- **ESLint** linter
+
+  `npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x`
+
+
+### Run linters
+- **Webhint** 
+linter -`npx hint .` on the root of your directory of your project.
+
+- **Stylelint** 
+linter - `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
+
+- **ESLint** linter
+  - `npx eslint .` on the root of your directory of your project.
+
+
+### Run project
+- Now run the following:
+ `npm install`  - on the root directory of the project to install the required packages
+ `npm start`   - on the root directory of the project to start the project 
+
+
+## Authors
+
+👤 **Richard Opiyo**
+
+- GitHub: [@richardoppiyo](https://github.com/richardoppiyo)
+- Twitter: [@blessed_ricky](https://twitter.com/blessed_ricky)
+- LinkedIn: [Richard Opiyo](https://linkedin.com/in/richardoppiyo)
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To morning session team,reviewers and standup team members.
+- Design by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] the link to the license is not working as expected, please have another look at [the template](https://github.com/microverseinc/readme-template) provided by Microverse for an example of how it is intended to work.